src/components/articleTeaser/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react';
+import { chakra, ResponsiveValue, useMultiStyleConfig } from '@chakra-ui/react';
+
+import Stack from '../stack';
+import { Sizes } from '../../themes';
+
+type Props = {
+  title: string;
+  text: string;
+  imageUrl: string;
+  url: string;
+  maxImgW?: ResponsiveValue<Sizes>;
+};
+
+const ArticleTeaser: FC<Props> = ({
+  title,
+  text,
+  imageUrl,
+  url,
+  maxImgW = '4xl',
+}) => {
+  const styles = useMultiStyleConfig('ArticleTeaser');
+
+  return (
+    <article>
+      <a href={url} target="_blank" rel="noopener noreferrer">
+        <Stack
+          direction={{
+            xs: 'column',
+            md: 'row',
+          }}
+          spacing={{ md: 'lg' }}
+          align="center"
+        >
+          <chakra.img maxW={maxImgW} src={imageUrl} />
```

Comment:
One question:
What is the reason behind using chakra factories and normal tags?

Reply:
Chakra factories are needed to properly consume theme values.

Reply:
True, I was confused with the parts into the tags. Thanks!